.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "nginx-configparser"]
+	path = nginx-configparser
```

Comment:
For this project, can we keep modules under a /src directory? So this path would be src/nginx-configparser. This way the top level can just have folders like src, lib, build, bin (et cetera), and modules can stay sequestered inside src.